script/run-unit-test.js

```diff
@@ -0,0 +1,37 @@
+const printUnitTestHelp = require('./run-unit-test-help.js');
+const commandLineArgs = require('command-line-args');
+const { runCommand } = require('./utils');
+
+const COMMAND_LINE_OPTIONS_DEFINITIONS = [
+  { name: 'log-level', type: String, defaultValue: 'log' },
+  { name: 'coverage', type: Boolean, defaultValue: false },
```

Comment:
I decided to add a switch for code coverage here, but I didn't replace the other bash script for now.